R/getWebServiceData.R

```diff
@@ -52,16 +52,15 @@ getWebServiceData <- function(obs_url, ...){
     
     headerInfo <- headers(returnedList)
 
-    if(headerInfo$`content-type` == "text/tab-separated-values;charset=UTF-8"){
+    if(headerInfo$`content-type` %in% c("text/tab-separated-values;charset=UTF-8")){
```

Comment:
Oh, I had initially put the json content-type in here, but after fiddling realized it needed to be downloaded like the zip....so, it could go back or stay, I don't think it matters